domains/GH9.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "GH9yt.is-a.dev",
+  "repo": "https://gh9youtube.github.io/GH9.github.io/",
+  "owner": {
+    "username": "GH9",
```

```suggestion
    "username": "GH9youtube",
```
username should be your github username